dspace-api/src/main/java/org/dspace/discovery/indexobject/ItemIndexFactoryImpl.java

```diff
@@ -605,6 +605,28 @@ public void addDiscoveryFields(SolrInputDocument doc, Context context, Item item
         }
 
 
+
```

Comment:
@milanmajchrak how comes this was required?
TT

Reply:
It is not Vanilla feature: `"_comp"`